packages/apps/rwa-demo/codegen-sdk.yml

```diff
@@ -1,6 +1,6 @@
 overwrite: true
-# schema: ./node_modules/@kadena/graph/generated-schema.graphql
-schema: https://graph.kadena.network/graphql
+schema: ./node_modules/@kadena/graph/generated-schema.graphql
+# schema: https://graph.kadena.network/graphql
```

Comment:
doesnt this need the kdaindexer schema as well?

Reply:
We'll first deploy explorer with kadindexer. Lateron we can modify this. The current changes are based on the diff between the two endpoints.

```sh
bunx @graphql-inspector/cli diff https://graph.kadena.network/graphql https://www.kadindexer.io/graphql > diff-kadena-hackachain-schema.log
```